packages/mdl-radio/mdl-radio.scss

```diff
@@ -0,0 +1,108 @@
+$mdl-radio-size: 20px !default;
+$mdl-radio-foreground-color: #333 !default;
+$mdl-radio-accent-color: rgba(255,82,82,0.87) !default;
+$mdl-radio-disabled-color: gray !default;
+$mdl-radio-timing-fn: ease 280ms !default;
+
+// Top-level host container.
+.mdl-radio {
+  color: $mdl-radio-foreground-color;
+  cursor: pointer;
+  display: inline-block;
+  transition: color $mdl-radio-timing-fn;
+}
+
+// Container for radio circles and ripple.
+.mdl-radio__container {
+  box-sizing: border-box;
+  display: inline-block;
+  height: $mdl-radio-size;
+  position: relative;
+  width: $mdl-radio-size;
+  top: 2px;
+}
+
+// The outer circle for the radio, always present.
+.mdl-radio__outer-circle {
+  border-color: $mdl-radio-foreground-color;
+  border: solid 2px;
+  border-radius: 50%;
+  box-sizing: border-box;
+  height: $mdl-radio-size;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transition: border-color $mdl-radio-timing-fn;
```

Comment:
we should add a `will-change` property here and to others with transition properties.